e2e/ansible/roles/k8s-custom-storageclass/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+
+---
+
+pool_name: pool1
+
+
+replica_count: 3
+
+
+tag: ci
```

Comment:
Does this be overwrite the one set in all.yml? Is the one specified in all.yml being used?

Reply:
Yes. This will override the value in global. No change to openebs-operator values.